boards.txt

```diff
@@ -16527,10 +16928,10 @@ deneyapkartg.menu.PartitionScheme.huge_app.upload.maximum_size=3145728
 deneyapkartg.menu.PartitionScheme.min_spiffs=Minimal SPIFFS (1.9MB APP with OTA/190KB SPIFFS)
 deneyapkartg.menu.PartitionScheme.min_spiffs.build.partitions=min_spiffs
 deneyapkartg.menu.PartitionScheme.min_spiffs.upload.maximum_size=1966080
-deneyapkartg.menu.PartitionScheme.fatflash=16M Flash (2MB APP/12.5MB FATFS)
+deneyapkartg.menu.PartitionScheme.fatflash=16M Flash (2MB APP/12.5MB FAT)
```

Comment:
As this is a standard partition scheme used on many boards, lets keep the naming the same please.